tests/test_xbog_calendar.py

```diff
@@ -0,0 +1,142 @@
+from unittest import TestCase
+import pandas as pd
+from pytz import UTC
+
+from .test_trading_calendar import ExchangeCalendarTestBase
+from trading_calendars.exchange_calendar_xbog import XBOGExchangeCalendar
+
+
+class XBOGCalendarTestCase(ExchangeCalendarTestBase, TestCase):
+
+    answer_key_filename = 'xbog'
+    calendar_class = XBOGExchangeCalendar
+
+    # The XBOG is open from 9:30AM to 4:00PM
+    MAX_SESSION_HOURS = 6.5
+
+    HAVE_EARLY_CLOSES = False
+
+    DAYLIGHT_SAVINGS_DATES = ['2018-03-12', '2018-11-06']
+
+    def test_regular_holidays(self):
+        all_sessions = self.calendar.all_sessions
+
+        expected_holidays = [
+            pd.Timestamp('2019-01-01', tz=UTC),  # New Year's Day
+            pd.Timestamp('2019-01-07', tz=UTC),  # Epiphany
+            pd.Timestamp('2019-03-25', tz=UTC),  # St. Joseph's Day
+            pd.Timestamp('2019-04-18', tz=UTC),  # Maundy Thursday
+            pd.Timestamp('2019-04-19', tz=UTC),  # Good Friday
+            pd.Timestamp('2019-05-01', tz=UTC),  # Labour Day
+            pd.Timestamp('2019-06-03', tz=UTC),  # Ascension Day
+            pd.Timestamp('2019-06-24', tz=UTC),  # Corpus Christi
+            pd.Timestamp('2019-07-01', tz=UTC),  # Sacred Heart
+            pd.Timestamp('2018-07-02', tz=UTC),  # St. Peter and St. Paul Day
+            pd.Timestamp('2018-07-20', tz=UTC),  # Colombian Independence Day
+            pd.Timestamp('2019-08-07', tz=UTC),  # Battle of Boyaca
+            pd.Timestamp('2019-08-19', tz=UTC),  # Assumption Day
+            pd.Timestamp('2019-10-14', tz=UTC),  # Dia de la Raza
+            pd.Timestamp('2019-11-04', tz=UTC),  # All Saint's Day
+            pd.Timestamp('2019-11-11', tz=UTC),  # Cartagena Independence Day
+            pd.Timestamp('2017-12-08', tz=UTC),  # Immaculate Conception
+            pd.Timestamp('2019-12-25', tz=UTC),  # Christmas Day
+            pd.Timestamp('2019-12-31', tz=UTC),  # Last Trading Day
```

Comment:
See my comment below, but I don't see this on https://en.bvc.com.co/pps/tibco/portalbvc/Home/Mercados/dianobursatil?action=dummy